packages/metro-config/src/ExpoMetroConfig.ts

```diff
@@ -223,7 +223,8 @@ export function getDefaultConfig(
     watchFolders,
     resolver: {
       resolverMainFields,
-      platforms: ['ios', 'android', 'native'],
+      platforms: ['ios', 'android', 'native', 'testing'],
+      assetExts: [...metroDefaultValues.resolver.assetExts.filter(assetExt => assetExt !== 'json')],
```

Comment:
Might we want to exclude anything in sourceExts from assetExts? That is, we'd make it a rule that any file that gets bundled by Metro is never treated as an asset by Metro.
(Also `filter()` returns an array already)
```suggestion
      assetExts: metroDefaultValues.resolver.assetExts.filter(assetExt => !sourceExtsSet.has(assetExt))],
```